### DISCUSSION TITLE
Two different approaches to constructing the single cell data frame give two different results

### DISCUSSION THREAD

**Discussion by AidanMar** [stuart-lab/signac#1032] [Q&A] [answered]
Hi there, currently using signac to analyse to scATAC data. I believe the standard way to read in the scATAC data output by cellranger-atac is described in this link here: [https://satijalab.org/signac/articles/pbmc_vignette.html](url). This does most of the heavy lifting by using the Read10X_h5 function. For some reason, my hpc does not allow .h5 files to be read. I have (really!) tried to fix that, but there appears to be no short term ability to do this. Instead, I tried to borrow the template from the merging objects vignette ([https://satijalab.org/signac/articles/merging.html](url)). However instead of running the final merge step, I am just running it on a single sample. I just follow all of the steps down to the "Create the objects" section, just using my single sample with no merger. This appears to work and gives me a seeminlgy valid seurat object. My only problem is that I end up with many more cells than anticapated. If I read outs/filtered_peak_bc_matrix/barcodes.tsv I get ~3,000, but if I create a Seurat object via this different approach I end up with roughly ~6,000 cells in that file. 

Is something wrong with this, or should I simply subset the seurat object based on the list of ~3,000 cells in the barcodes.tsv?

Any help would be immensely appreciated. This is such a great tool to use!

**Comment by timoast** [comment, marked as answer]
Cellranger uses a different method of determining the cutoff for keeping cell barcodes. In the Signac vignettes, we tend to look at the total fragments per cell and QC metrics like TSS enrichment and nucleosome signal. You'll need to decide on QC cutoffs that make sense given the distribution of values for your specific dataset, and if you're retaining too many cells (especially if you don't expect that many cells from the experiment), you may need to increase the cutoffs used.
